src/helpers/api.ts

```diff
@@ -25,6 +25,11 @@ export async function fetchBlocksData(limit = 30, sort = 'desc', page = 0): Prom
     if (sort === 'desc') {
         blocks.blocks.sort((a, b) => b.block.header.height - a.block.header.height);
     }
+
+    if(blocks.blocks.length > config.maxBlocks) {
+        blocks.blocks.splice(-1,1);
+    }
```

Comment:
Move this to the redux action. Also how do we know that we only need to reduce by 1? We could've fetched 200 blocks.